language-server/src/server.js

```diff
@@ -148,43 +179,70 @@ documents.onDidChangeContent(async ({ document }) => {
 
 const validateSchema = async (document) => {
   const diagnostics = [];
-
+  const settings = await getDocumentSettings(document.uri);
+  connection.console.log(JSON.stringify(settings));
```

Comment:
While I'm sure this was useful in debugging, it should be removed before this is merged.

```suggestion
```